frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx

```diff
@@ -42,22 +41,18 @@ const StyledButtonSection = styled.div`
 `
 
 export function MissionQueueCard({ mission, onDeleteMission }: MissionQueueCardProps) {
-    let navigate = useNavigate()
-    const routeChange = () => {
-        let path = `${config.FRONTEND_BASE_ROUTE}/mission/${mission.id}`
-        navigate(path)
-    }
+    const missionLink = `${config.FRONTEND_BASE_ROUTE}/mission/${mission.id}`
     const [confirmDeleteDialogOpen, setConfirmDeleteDialogOpen] = useState<boolean>(false)
     var numberOfTasks = 0
     mission.tasks.forEach((task) => (numberOfTasks += task.inspections.length))
     return (
         <StyledMissionCard key={mission.id} variant="default" style={{ boxShadow: tokens.elevation.raised }}>
             <HorizontalContent>
-                <HorizontalNonButtonContent onClick={routeChange}>
+                <HorizontalNonButtonContent>
                     <Checkbox />
-                    <Button variant="ghost" fullWidth>
-                        <Typography variant="body_short_bold">{mission.name}</Typography>
-                    </Button>
+                    <Typography link href={missionLink}>
+                        {mission.name}
+                    </Typography>
```

Comment:
I think I prefer the old design over the new

Old design:
![image](https://user-images.githubusercontent.com/33518988/229138503-a83e7e9c-98e9-4a83-b4a2-11d475ad0517.png)

New design:
![image](https://user-images.githubusercontent.com/33518988/229138307-43ff3c30-fd73-442a-a64c-d011556b748c.png)